src/Components/Members/Create/index.jsx

```diff
@@ -0,0 +1,234 @@
+    e.preventDefault();
+    addMember({
+      firstName,
+      lastName,
+      dni,
+      phone,
+      email,
+      city,
+      birthDay,
+      postalCode,
+      isActive,
+      membership
+    });
+    setFirstName('');
+    setLastName('');
+    setDni('');
+    setPhone('');
+    setEmail('');
+    setCity('');
+    setBirthday('');
+    setPostalCode('');
+    setIsActive(true);
+    setMembership('Classic');
+    setShowModal(true);
+  };
+
+  return (
+    <div className={styles.formContainer}>
+      <Modal
+        onClose={() => setShowModal(false)}
+        isOpen={showModal}
+        title={message}
+        success={true}
+      />
+      ;
+      <div>
+        <h3>Create new member</h3>
+        <form onSubmit={(e) => handleSubmit(e)}>
+          <div>
+            <Input
+              labelText={'Name'}
+              type={'text'}
+              placeholder={'Name'}
+              value={firstName}
+              onChangeInput={handleFirstName}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'Surname'}
+              type={'text'}
+              placeholder={'Surname'}
+              value={lastName}
+              onChangeInput={handleLastName}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'DNI'}
+              type={'text'}
+              placeholder={'DNI'}
+              value={dni}
+              onChangeInput={handleDniChange}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'Phone'}
+              type={'text'}
+              placeholder={'ex: 096513178'}
+              value={phone}
+              onChangeInput={handlePhoneChange}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'Email'}
+              type={'text'}
+              placeholder={'robertomariaoverdrive@soybostero.edu'}
+              value={email}
+              onChangeInput={handleEmailChange}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'City'}
+              type={'text'}
+              placeholder={'Your city'}
+              value={city}
+              onChangeInput={handleCityChange}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'Birthday'}
+              type={'text'}
+              placeholder={'01/01/2000'}
+              value={birthDay}
+              onChangeInput={handleBirthdayChange}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'Zip Code'}
+              type={'text'}
+              placeholder={'Your postal code'}
+              value={postalCode}
+              onChangeInput={handlePostalCodeChange}
+            />
+          </div>
+          <div>
+            <label>
+              Membership:
+              <select onChange={handleMembershipChange}>
```

Comment:
Yes, I can see that. However, the selected component won't be able to map correctly the membership types, since they are not on the database. In order to use the shared component I would have to hardcode a lot of data in order to make it "fit" onto the seletct shared input, therefore, I figured it was better to use a regular selector instead.